### PR TITLE
chore: install always last version of MSW

### PR DESCRIPTION
**Pull request by marcosvega91**
When we run test script I think that we always want to use the latest published version of MSW.

This will also solve the problem of dump dependency when a new version is released.

### CODE REVIEW

File: examples/rest-react/README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This repository illustrates how to use [Mock Service Worker](https://github.com/
 $ git clone https://github.com/mswjs/examples.git
 $ cd examples
 $ yarn
+$ yarn update
```
Comment (kettanaito) [inline examples/rest-react/README.md:18]: I understand the intention here, but `yarn update` is an internal command to keep the worker script up-to-date. I think it's not a good idea to suggest to run it to our users in these "Getting started" instructions. 

These instructions should represent all they have to do to run the project locally, and asking them to update or generate workers is redundant, as we recommend committing the worker script to Git. I think we should remove this command and keep the worker script updated by our means (perhaps, more automated then now, but definitely not via the expense of asking our users to do so).

Comment (marcosvega91) [reply examples/rest-react/README.md:18]: If we use postinstall instead ?

Comment (kettanaito) [reply examples/rest-react/README.md:18]: `postinstall` is a good compromise. Yet I think it's really useful if the entire state of the repo in Git represents what you, as the end developer, should have. That includes the `mockServiceWorker.js` file in the right directory.

It'd be harder for people to understand what's wrong if we omit certain part of the setup and implicitly add it later on as a `postinstall` script. 

What is the issue you see with the committed worker script?

Comment (marcosvega91) [reply examples/rest-react/README.md:18]: Yes what you said is right, and I understand the problem. 
Committing `mockServiceWorker.js` is not a problem of course, but there will be cases where it is changed. 

But I agree with you that the user using examples should have everything

Comment (kettanaito) [reply examples/rest-react/README.md:18]: The worker script is updated as a part of the smoke test script, so we always run smoke tests against the latest `msw`.

The repository, however, is to the manual maintenance, which makes sense and gives a transparency whenever we make a change.

Comment (marcosvega91) [reply examples/rest-react/README.md]: I have added again all service workers 😄

**Pull request merged**